### PR TITLE
mosdepth: 0.3.1 -> 0.3.2

### PR DESCRIPTION
**Pull request by jbedo**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/science/biology/mosdepth/default.nix [hunk 1]
```diff
@@ -17,13 +17,13 @@ let
...
   pname = "mosdepth";
-  version = "0.3.1";
+  version = "0.3.2";
 
   src = fetchFromGitHub {
     owner = "brentp";
     repo = "mosdepth";
     rev = "v${version}";
-    sha256 = "1kcrvamrafz1m0s7mlbhaay8jyg97l1w37p6syl36r2m1plmwxjd";
+    sha256 = "sha256-uui4yC7ok+pvbXVKfBVsAarH40fnH4fnP8P4uzOqztQ=";
```
Comment (Kreyren) [inline pkgs/applications/science/biology/mosdepth/default.nix:26]: Do you have to specify the `sha256-` here if it's already in the `sha256` ?

**Comment by jbedo** [comment]
No, but my understanding is this is the currently accepted form.

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 131627` at d8853734 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built successfully:</summary>
  <ul>
    <li>mosdepth</li>
  </ul>
</details>
<details>
  <summary>2 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md">missing-phase-hooks</a>

`installPhase` should probably contain `runHook preInstall` and `runHook postInstall`.

Near pkgs/applications/science/biology/mosdepth/default.nix:37:3:
```
   |
37 |   installPhase = "install -Dt $out/bin mosdepth";
   |   ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md">missing-phase-hooks</a>

`buildPhase` should probably contain `runHook preBuild` and `runHook postBuild`.

Near pkgs/applications/science/biology/mosdepth/default.nix:32:3:
```
   |
32 |   buildPhase = ''
   |   ^
```
</li>
  </ul>
</details>

File: pkgs/applications/science/biology/mosdepth/default.nix (hunk 1 above)
Comment (risicle) [reply pkgs/applications/science/biology/mosdepth/default.nix:26]: This is the backwards compatible way of using a new format I think we're transitioning to.

Comment (SuperSandro2000) [reply pkgs/applications/science/biology/mosdepth/default.nix:26]: that is fine for now.

**Pull request merged**